ui/admin/app/components/scope-picker/index.hbs

```diff
@@ -0,0 +1,100 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<Hds::Dropdown
+  @enableCollisionDetection={{true}}
+  @matchToggleWidth={{true}}
+  class='hds-side-nav-hide-when-minimized scope-picker'
```

Comment:
[link to explanation of this hds class usage](https://helios.hashicorp.design/components/side-nav?tab=code#content-transitions-between-states)

Reply:
non-blocking: wonder what the norm is for commenting code? Since this class is the first instance in our codebase, wondering what you think about adding a comment for future readers, like:
```
{{! apply this special class to automatically fade in/out }}
```

Reply:
Yup i can add a comment. The norm is to add comments in hard to understand areas or where intended behavior is unclear & to help with readability. Other areas we add comments is to functions (class methods, getters, setters).